game/scripts/npc/heroes/obsidian_destroyer.txt

```diff
@@ -8,5 +8,9 @@
     // Abilities
     //-------------------------------------------------------------------------------------------------------------
     "Ability1"    "obsidian_destroyer_arcane_orb_oaa" // replaces obsidian_destroyer_arcane_orb
+
+    //Attributes
+    //-------------------------------------------------------------------------------------------------------------
+    "AttributeAgilityGain"		"2.400000"
```

Comment:
I would remove this as this is an unnecessary change.

Reply:
Attack speed hero in OAA which isn't an agi hero. This is why

Reply:
I'd rather you just reduce his BAT by 1 than. .4 agility x 50 is 20 extra attack speed at max level. That means almost nothing tbh